web_ui/src/pages/annotator/annotation/annotation-list/annotation-list.component.tsx

```diff
@@ -110,48 +69,27 @@ export const AnnotationList = ({ annotations, isLoading, isDragDisabled }: Annot
         );
     }
 
-    // We use react beautiful dnd in combination with virtual rendering via Virtuoso to make the
-    // annotation list both draggable and virtual.
-    // For more information and examples see:
-    // https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/patterns/virtual-lists.md
-    // https://virtuoso.dev/react-beautiful-dnd/
     return (
-        <DragDropContext onDragEnd={handleDragEnd}>
-            <Droppable droppableId='annotation-droppable-id' mode='virtual' renderClone={renderClone}>
-                {(provided: DroppableProvided) => {
-                    return (
-                        <Virtuoso<Annotation>
-                            role='list'
-                            aria-label='Annotations list'
-                            components={{ Item: HeightPreservingItem }}
-                            id='annotation-list'
-                            // @ts-expect-error innerRef does not contain Window, which is fine
-                            scrollerRef={provided.innerRef}
-                            data={reversedAnnotations}
-                            itemContent={(index, annotation) => {
-                                return (
-                                    <Draggable
-                                        draggableId={annotation.id}
-                                        index={index}
-                                        key={annotation.id}
-                                        isDragDisabled={isSceneBusy || isDragDisabled}
-                                    >
-                                        {(draggableProvided) => (
-                                            <DraggableAnnotation
-                                                provided={draggableProvided}
-                                                annotation={annotation}
-                                                isDragging={false}
-                                                isLast={index === reversedAnnotations.length - 1}
-                                                annotationToolContext={annotationToolContext}
-                                            />
-                                        )}
-                                    </Draggable>
-                                );
-                            }}
+        // "rowHeight" is necessary for testing purposes, or the container will render empty
```

Comment:
if it's just for testing purposes, can we not use the same approach we did with react-virtuoso and add a mock to the setupTests with this height?

Reply:
Excellent point. I tried it and it works. I’ll push it once I fix the component test issue

Reply:
https://github.com/open-edge-platform/geti/pull/144/commits/33dd012b54dfa32549e4ab761fa13fb863d1afef#diff-1e070bf0839c580e5b9a080d01dc105a7b85581ae431f7ccc2bbcdf31e20b527R105